notebooks/growth_model_classification.ipynb

```diff
@@ -0,0 +1,1053 @@
+{
```

Comment:
<p>Let's remove this cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>